src/rdev.rs

```diff
@@ -187,7 +187,7 @@ pub enum EventType {
 /// on the OS layout and keyboard state machinery.
 /// Caveat: Dead keys don't function on Linux(X11) yet. You will receive None for
 /// a dead key, and the raw letter instead of accentuated letter instead.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Why not copy here ?

Reply:
Because `Event` (could) contain a `String`, which is not `Copy`. The standard library made the choice to make `String` not `Copy` because a `String` could be really big (`usize:max_value()` bytes big), in which case copying it takes a long time. The convention around Copy/Clone is that if it could take an arbitrary amount of time to make a copy, implement Clone only.

Reply:
Not to mention that Rust won't let you `#[derive(Copy)]` because not all fields implement Copy. (you could still implement it manually)

Reply:
Fair enough. 

For the test that does not pass I don't really understand what's happening. I guess github macos runner changed and now respects accessibility ?